k9mail/src/test/java/com/fsck/k9/helper/HtmlSanitizerTest.java

```diff
@@ -99,4 +99,31 @@ public void shouldRemoveMetaRefreshButKeepOtherMetaTags() {
         assertEquals("<html><head><meta http-equiv=\"content-type\" content=\"text/html; charset=UTF-8\" /></head>" +
                 "<body>Message</body></html>", htmlSanitizer.sanitize(html));
     }
+
+    @Test
+    public void shouldProduceValidHtmlFromHtmlWithXmlDeclaration() {
+        String html = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
+                "<html><head></head><body></body></html>";
+        assertEquals("<html><head></head><body></body></html>", htmlSanitizer.sanitize(html));
```

Comment:
Please follow the arrange-act-assert structure. If you're concerned with consistency add another commit to rewrite the existing tests.